content/en/docs/reference/instrumentation/node-metrics.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Node metrics data
+content_type: reference
+weight: 50
+description: >-
+  Mechanisms for accessing metrics at node, volume, pod and container level,
+  as seen by the kubelet.
+---
+
+The [kubelet](/docs/reference/command-line-tools-reference/kubelet/)
+gathers metric statistics at the node, volume, pod and container level,
+and emits this information in the
+[Summary API](https://github.com/kubernetes/kubernetes/blob/7d309e0104fedb57280b261e5677d919cb2a0e2d/staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go).
```

Comment:
We can add a reference config API for this. The markdown is there, not published.

Reply:
Sounds like a nice improvement to make.